src/logo/builtin.c

```diff
@@ -3105,6 +3105,16 @@ static const FFlogo M[] = {
         .colorKeys = FF_COLOR_FG_LIGHT_BLACK,
         .colorTitle = FF_COLOR_FG_DEFAULT,
     },
+    // Minimal System
+    {
+	.names = {"Minimal System", "Minimal"},
```

Comment:
```suggestion
	.names = {"Minimal_System"},
```

To match `ID` in your `/etc/os-release`